HLTrigger/Configuration/scripts/hltIntegrationTests

```diff
@@ -241,7 +241,7 @@ mkdir  "$WORKDIR"
 cd     "$WORKDIR"
 
 # find the list of all trigger paths
-TRIGGERS=$(hltListPaths "$MENU" $HLTLISTPATHPROXY -p --no-dep --exclude "^HLTriggerFinalPath$" "^Dataset_.+" "^DST_HLTMuon_Run3_PFScoutingPixelTracking_v[0-9]+$")
```

Comment:
I realised that these hard-coded names can be removed here; specifying `--no-dep` is sufficient to omit Paths that depend on other Paths.